### PR TITLE
Adds Parents to Devotionals and Sermons

### PR DESCRIPTION
**Pull request by redreceipt**
## DESCRIPTION

Screen Shot 2019-08-28 at 12 45 30 PM

### What does this PR do, or why is it needed?

Adds a `series` field to sermon and devotional content items to get their parent series content items.

### What design trade-offs/decisions were made?

None.

### How do I test this PR?

Run this:

```
{
  userFeed {
    edges {
      node {
        ... on WeekendContentItem {
          id
          series {
            title
          }
        }
        ... on DevotionalContentItem {
          id
          series {
            title
          }
        }
      }
    }
  }
}
```

### What automated tests did you write?

None.

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [#1023]
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by redreceipt** [comment]
Got it. I didn't realize the different `ContentItem` types had their own data source functions.

File: packages/apollos-church-api/config.yml [hunk 1]
```diff
@@ -108,6 +108,8 @@ ROCK_MAPPINGS:
     - 11
 
   SERMON_CHANNEL_ID: 5
+  SERMON_SERIES_CHANNEL_ID: 4
+  DEVOTIONAL_SERIES_CHANNEL_ID: 6
```
Comment (conrad-vanl) [inline packages/apollos-church-api/config.yml]: I'd like @vinnyjth's feedback on this config syntax. I wonder if there's a more flexible approach here that would be just as easy?

Perhaps something like a list of channel IDs and their parent channel ID?

So like content in channel ID `4` - use channel ID `6` for that series parent

**Comment by richarddubay** [comment]
@conrad-vanl @vinnyjth Any other thoughts on this one? The use case for this is adding the series to the sermon notes when they are exported.

**Comment by redreceipt** [comment]
core's use case is displaying series metadata on individual sermon content items

**Comment by redreceipt** [comment]
I've moved this over to our repo, we can close this if it doesn't belong in core.

**Pull request closed without merging**